src/components/inputs/Pay/MemoFormInput.tsx

```diff
@@ -44,7 +45,9 @@ export function MemoFormInput({
           }}
         >
           <Tooltip title={t`Attach a sticker`}>
```

Comment:
**suggestion** I think we should remove the tooltip. feels distracting (i know its unrelated to your PR)

Reply:
https://github.com/jbx-protocol/juice-interface/pull/1506/files -- addressed in another PR since it's unrelated.

Reply:
agree, it's super distracting.